### ISSUE TITLE
DEV:LOG: Maintenance (missing return information)

### ISSUE THREAD

**Issue by TessForGithub2** [hashmapinc/Drillflow#459]
Size: 
Small

QA reports the following information is "missing" retrieving information that was supposedly added (per success return code) as part of the verification step.

1. indexCurve
    It is converted to ChannelIndex.mnemonic as documented in the Mapping document.
    This one is closed.
2. classWitsml
    Verified via code trace that this is not sent over to DoT (although it used to be sent over).
    This is still open pending my trace as to why this is happening.
3. stepIncrement
    It is sent to DoT (verified via code trace) and viewed with "GET ChannelSet Metadata" 
    after AddToStore. This one is closed.
4. curveDescription
    This becomes ChannelModel.description per Mapping document. I will trace one more
    time, but believe this one is closed.

In summary, I think only 2 above is still in question & in need of code tracing/maintenance.

**Comment by Mike-d-s** [comment]
EPIC:
#361 Log CRUD Implementation

**Comment by shehzadsidi** [comment]
classWitsml and curveDescription still missing in the response (Log Depth. and Time for 1.3.1.1 and 1.4.1.1)

**Comment by niaalex** [comment]
New ETA for this item 6/17. @TessForGithub2

**Comment by TessForGithub2** [comment]
UPDATE: classWitsml & curveDescription are now fixed & tested. 

Fix placed into df-551 for speed of testing.
